public/app/redux/redux-query-sync.ts

```diff
@@ -0,0 +1,109 @@
+import { actions as tracingActions } from '@pyroscope/redux/reducers/tracing';
+import { history } from '@pyroscope/util/history';
+import ReduxQuerySync from 'redux-query-sync';
+import { actions as continuousActions } from './reducers/continuous';
+import { RootState, StoreType } from './store';
+
+let unsubscribeFromReduxQuerySync: unknown;
```

Comment:
I am not expecting that this should appear outside of react. Maybe we can make this into a hook, and the useEffect in the hook here can keep the `unsubscribeFromReduxQuerySync` and return it.

Then the `App` just needs to use the hook.

Reply:
Good point, I'll make the changes!